cmd/rekor_monitor/main.go

```diff
@@ -226,16 +242,34 @@ func mainLoopV2(tufClient *tuf.Client, flags *cmd.MonitorFlags, config *notifica
 				}
 			}
 
-			// TODO: should return prev and cur
-			cur, err := rekor_v2.RunConsistencyCheck(context.Background(), rekorShards, latestShardOrigin, flags.LogInfoFile)
+			prev, cur, err := rekor_v2.RunConsistencyCheck(context.Background(), rekorShards, latestShardOrigin, flags.LogInfoFile)
```

Comment:
@facutuesca I think this was wrong, as `rekor_v2.RunConsistencyCheck` was returning just the previous checkpoint and not the current one, however here we were considering it as the current one.

Correct? Please have a deeper look to ensure what I did make sense.